authors/apps/authentication/serializers.py

```diff
@@ -15,6 +17,16 @@ class RegistrationSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
+    def validate(self, data):
+        username = data.get('username', None)
+        email = data.get('email', None)
+        password = data.get("password", None)
+
+        validator = ValidateRegistrationData()
+        validator.password_validation(password)
+        validator.username_validation(username)
+        return {'email': email, 'username': username, 'password': password}
```

Comment:
@myrdstom I think you should do as suggested by @fahadmak,i understands it add more line this would  make your code more readable.